app/files.test.js

```diff
@@ -207,4 +208,20 @@ Files("triggers when a room is archived", async (context) => {
   not.equal(room.id, 0)
 })
 
+Files("triggers when a folder is deleted", async (context) => {
+  const { perform } = folderDeleted.operation
+  // TODO: Add a check to delete a folder from a specific room after adding a Delete Folder action
+  /*const inputData = {
+    id: context.inputData.folderId
+  }*/
+  const bundle = {
+    authData: context.authData
+  }
+  const folders = await tester(perform, bundle)
+  const folder = folders[0]
+  // TODO: Without the folder delete action, the folder delete trigger test may fail because the trash may be empty.
+  if (folders.length) not.equal(folder.id, 0)
+  else equal(true, true)
```

Comment:
Please avoid using one-liners as they can be difficult to read. To help with this, I suggest adding an eslint rule to enforce this guideline.

Reply:
> Please avoid using one-liners as they can be difficult to read. To help with this, I suggest adding an eslint rule to enforce this guideline.

Corrected. The eslint rule will be added in a separate PR.